gammapy/utils/modeling.py

```diff
@@ -365,11 +423,11 @@ class SpectralModelPowerLaw(SpectralModel):
 
     @classmethod
     def from_pset(cls, pset):
```

Comment:
Since it's not called `ParameterList`, all the "pset" should be changed to "plist", no?
Can you do it here or should I in a follow-up commit?